admin_manual/release_notes.rst

```diff
@@ -359,12 +360,10 @@ Protecting ownCloud on IIS from Data Loss
 Under certain circumstances, running your ownCloud server on IIS could be at 
 risk of data loss. To prevent this, follow these steps.
 
-In your ownCloud server configuration file, ``owncloud\config\config.php``, set 
+* In your ownCloud server configuration file, ``owncloud\config\config.php``, set 
```

Comment:
Not sure if this is working as expected. This is broken in stable9.1+ as seen on https://doc.owncloud.org/server/9.1/admin_manual/release_notes.html#protecting-owncloud-on-iis-from-data-loss